.github/workflows/sync-releases-with-main.yml

```diff
@@ -0,0 +1,31 @@
+name: Sync releases with main
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+
+jobs:
+  sync-release-branches:
+    strategy:
+      matrix:
+        version: ["23.11", "24.05"]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: "release-${{ matrix.version }}"
+    - run: |
+        git config --global merge.ours.driver true
+        git config --global user.name 'autofirma-nix-bot'
+        git config --global user.email 'autofirma-nix-bot@users.noreply.github.com'
+        git config --global pull.rebase false
+    - run: |
+        git fetch origin main
+        git pull --allow-unrelated-histories origin main:main
+    - run: git merge main
```

Comment:
Unless I'm missing something, this merge is redundant. I would leave a single `git merge --allow-unrelated-histories origin main:main` replacing this and the previous line.